action.yml

```diff
@@ -1,5 +1,5 @@
-name: asciidoctor-ghpages
-author: Manoel Campos da Silva Filho
+name: asciidoctor-ghpages-lfs
```

Comment:
I don't see the need to change the name of the workflow and these metadata. Maybe you have changed to test the workflow in your fork.